src/constans.ts

```diff
@@ -2,3 +2,27 @@ export const currencies = [
 	{ value: "EN", label: "English" },
 	{ value: "DE", label: "Deutsch" },
 ];
+
+export const footerData = [
+	"Häufig gestellte Fragen (FAQ)",
+	"Konto",
+	"Karriere",
+	"Geschenkkarten kaufen",
+	"Datenschutz",
+	"Kontakt",
+	"Rechtliche Hinweise",
+	"Mitglidschaft Kündigen",
+	"Medien-Center",
+	"Netflix-Schop",
+	"So Könne Sie Netflix ansehen",
+	"Cookies-Einstellungen",
+	"Geschwindigkeitstest",
+	"Nur auf Netflix",
+	"Hilfe-Center",
+	"Anlegerbeziehungen",
+	"Geschebnkkarten einlösen",
+	"Nutzungbedingungen",
+	"Impressum",
+	"Gesetzliche Garantie",
+	"Wahlmöglichkeiten für Werbung",
+];
```

Comment:
Te dane są okej jako "napis na linku" natomiast fatalne jeśli chodzi o URL. Opcje są dwie:
- Albo tutaj robisz tablicę obiektów i każdy element ma label oraz url (url w formacie z myslnikami, male litery bez specjalnych znaków
- ALBO definiujesz funkcję/sciągasz bibliotekę która zamienia podany string w formacie czytelnym na format URL z myslnikami, lowercase itd. 

Nie możemy mieć linka `Häufig gestellte Fragen (FAQ)` - to musi być /faq przykładowo, z małej litery, jesli sa spacje to w postaci myslnika.